schema.sql

```diff
@@ -11,3 +11,55 @@ CREATE TABLE animals (
 -- Adding new table header
 ALTER TABLE animals
 ADD COLUMN species VARCHAR(100);
+
+---------------------------------------------------------------------------------------------
+-- Create a table named owners with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- full_name: string
+-- age: integer
+
+CREATE TABLE owners (
+    owner_id SERIAL PRIMARY KEY,
+    full_name VARCHAR(100),
+    age INT
+);
+
+---------------------------------------------------------------------------------------------
+-- Create a table named species with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+
+CREATE TABLE species (
+    species_id SERIAL PRIMARY KEY,
+    name VARCHAR(100)
+);
```

Comment:
- [ ] It would be nice if you rename this `species_id` to `id` because when you make a table and you want an id for that table then you never use the name of the table in the id column except you are referencing that column as a foreign key but even that comes in the table which is child table it is a standard of SQL and it is mentioned in the project requirements 🤔 Kindly make this change to make your project follow the Standards of SQL and project requirements 👍